src/reanimated2/UpdateProps.js

```diff
@@ -23,8 +23,13 @@ const colorProps = [
 
 const ColorProperties = makeShareable(colorProps);
 
-export default function updateProps(viewTag, updates, maybeViewRef) {
+export default function updateProps(viewTag, viewName, updates, maybeViewRef) {
   'worklet';
+  if (viewName && viewName.value) {
+    viewName = viewName.value;
+  } else {
+    viewName = 'RCTView';
+  }
```

Comment:
`viewName = viewName?.value ?? 'RCTView';` should be semantically equivalent to this if. BTW, this reassign forces update, right?

Reply:
> this reassign forces update, right?
What update? What do you mean?

Reply:
Resolved internally.